zcash_primitives/src/transaction/tests/mod.rs

```diff
@@ -1,1921 +1,27 @@
-use pairing::bls12_381::Bls12;
-use rand::{thread_rng, Rng};
-use sapling_crypto::{jubjub::FixedGenerators, redjubjub::PrivateKey};
-
 use super::{
     components::{Amount, Script},
     sighash::signature_hash,
-    Transaction, TransactionData,
+    Transaction,
 };
-use JUBJUB;
+
+mod data;
+
 
 #[test]
 fn tx_read_write() {
     // TxID: 64f0bd7fe30ce23753358fe3a2dc835b8fba9c0274c4e2c54a6f73114cb55639
     // From testnet block 280003.
```

Comment:
Move these two comment lines as well; they belong with the test data rather than the test.